src/vs/editor/common/config/editorOptions.ts

```diff
@@ -496,6 +496,10 @@ export interface IEditorOptions {
 	 * Control the behavior and rendering of the code action lightbulb.
 	 */
 	lightbulb?: IEditorLightbulbOptions;
+	/**
+	 * Code action kinds to be run on save.
+	 */
+	codeActionsOnSave?: string[];
```

Comment:
Could be an object literal because that enables merging/overwriting

Reply:
Done. Yes an object is much nicer to work with